scenarios2/10_Lilith.cfg

```diff
@@ -2052,6 +2052,31 @@ As the time comes for two suns to shine brightly in the sky."
             message= _ "Not until I allow it! This should prevent her from dying. We will worry about her later."
         [/message]
     [/event]
+    [event]
+        name=last breath
+        [filter]
+            id=Corruptia
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "No! Save me! Curse you, Lilith!"
+        [/message]
+        [message]
+            speaker=Efraim
+            message= _ "Among the leaders of Arach she was the only one who turned away from Lilith. She might be helpful to bring local people back to normal life."
```

Comment:
_From all of Arach's council members, she was the only one who turned away from Lilith. She might turn out to be helpful at bringing normal life to the city._